src/Microsoft.Health.Fhir.Core/Resources.Designer.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Microsoft.Health.Fhir.Core {
     using System;
-
-
+    
+    
```

Comment:
Super nit: is there an easy way to avoid adding these whitespaces?

Reply:
It's a fight between VS Studio and Rider, I'm not willing to participate in it.